administrate/metrics/overview.md

```diff
@@ -432,17 +433,65 @@ Below, the list of all warp10 classes representing Telegraf metrics :
 <tr>
 <td align="center">mem.swap_cached</td>
 </tr>
+<tr>  <td align=”center”> jvm.statsd-jvm-profiler_heap_ps-old-gen_max.value</td>
```

Comment:
Those are specific to the JVM, maybe we could add some sort of categories to split them? We could later add them specific to PHP and such.

Reply:
Okay, that's better, indeed. 

Which are the others missing-in-the-doc specific classes  ?

Reply:
There are quite a lot of them. There are varnish stats, apache stats, and for all the various add-ons. That's a lot.

I'm not really sure that's a good idea to expose all of them in the documentation as they may be updated or new classes may appear.

Instead, we could maybe document on how to list them all?

Reply:
@vballu Do you need help on this ?

Reply:
@cnivolle , Not really. It is mainly a matter of times. I will update this doc when the number of GTS will be reduce

Reply:
@vballu I can take over this PR if you want